### PR TITLE
Raid Share & Vote Update 

### PR DESCRIPTION
**Pull request by studplus**
- Updates all Raid Polls in different Channels
- New Feature: If you are using multiple Channel, you can setup one Channel as Main-Channel and on votes in different Channel, the Raid will be shared to your Main-Channel.
This is important for Raids from Webhooks. All Raids were posted to one Channel, which can be muted to the users. But if someone votes for a raid, this raid will be posted to a unmuted channel, where all others get a notification.

### CODE REVIEW

File: mods/vote_time.php [hunk 1]
```diff
@@ -87,6 +122,65 @@
...
+                    g.gym_name AS gym_name,
+                    g.ex_gym AS ex_gym,
+                    g.show_gym AS show_gym,
+                    g.gym_note AS gym_note,
+                    g.gym_id AS gym_id2,
+                    g.img_url AS img_url           
+                FROM raids as r 
+                left join gyms as g on r.gym_id = g.id 
+                WHERE r.id = {$data['id']}
+                "
```
Comment (Chefkeks) [inline mods/vote_time.php:158]: Maybe move to own function? 🤔

File: mods/vote_time.php [hunk 2]
```diff
@@ -87,6 +122,65 @@
...
+                $tg_json[] = send_message($chat, $text['full'], $keys, ['reply_to_message_id' => $reply_to, 'reply_markup' => ['selective' => true, 'one_time_keyboard' => true], 'disable_web_page_preview' => 'true'], true);
+            }
+            // Telegram multicurl request.
+            curl_json_multi_request($tg_json);
+            // Update new entry
+            /*if ($config->RAID_PICTURE){
+                send_response_vote($update, $answer_gym,false,false); 
+             } else {
+                send_response_vote($update, $answer_gym); 
+             }*/
```
Comment (Chefkeks) [inline mods/vote_time.php:182]: Can this be removed or is it needed?
Was it for testing something or does it need to be un-comment?

**Comment by Chefkeks** [comment]
Nice :)
Just had 2 little questions, but overall it looks good to me.

File: mods/vote_time.php (hunk 2 above)
Comment (studplus) [reply mods/vote_time.php]: was coding/testing stuff before I had get_all_active_raid_channels in send_response_vote.php 
its deleted

File: mods/vote_time.php (hunk 1 above)
Comment (studplus) [reply mods/vote_time.php]: yes, done 👍

**Comment by studplus** [comment]
Ready for merge!?

**Comment by Chefkeks** [comment]
@studplus @Artanicus
I know it’s already merged, but shouldn’t we move the function out of mods/vote_time.php to a logic file ?

**Pull request merged**